### DISCUSSION TITLE
The system doesn't accept my info 🙁

### DISCUSSION THREAD

**Discussion by yxzvl** [community/community#56176] [GitHub Education]
### Hi everyone,

Hello!, I have a "big" problem with the tests that I sent to verify that I am a student. I pointed out that the proof I sent is a student ID, it contains the name of my school (obviously) and my full name too, I made sure that it contains everything they ask for (they only ask for the name of my school and my name) , despite the fact that the image contains everything, it simply gives me an error asking me to put what I already attached. If you can help me I would greatly appreciate it :).
<img width="754" alt="Captura de pantalla 2023-05-22 a la(s) 19 50 18" src="https://github.com/community/community/assets/112527596/01ba099e-a349-4f1b-9bd2-ffb269f584bc">

**Comment by samus-aran** [comment]
Hi @yxzvl 

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Regarding to School name we don't accept School logos and you should ensure that the school you selected matches what text is on the the submitted evidence. You state that the image contains the school name, but you must see if it uses a variation of the school or have different spelling.

Hope this helps 👍
